### ISSUE TITLE
Slider: Specify a decimal value for valuenow, valuemin, valuemax

### ISSUE THREAD

**Issue by 0ddfell0w** [w3c/aria#706]
The spec says that aria-valuenow, aria-valuemin and aria-valuemax should be numbers. The [Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.1/#Listbox) are more specific, requiring a decimal number.

Would it be possible to adopt the more specific "decimal" requirement in the spec? Thanks!

**Comment by carmacleod** [comment]
Agreed. It's pretty annoying to listen to a progressbar say 54.2287598147613875087451987 percent.  :)

**Comment by mcking65** [comment]
Section [6.2.4 Value](http://w3c.github.io/aria/#propcharacteristic_value) defines integer and number types:

>integer: A numerical value without a fractional component.
>number: Any real numerical value.

The APG guidance is based on this paragraph in the [aria-valuenow specification](http://w3c.github.io/aria/#aria-valuenow):

>The value of aria-valuenow is a decimal number. If the range is a set of numeric values, then aria-valuenow is one of those values. For example, if the range is [0, 1], a valid aria-valuenow is 0.5. A value outside the range, such as -2.5 or 1.1, is invalid.

Personally, I don't know why it always has to be a decimal. In many cases, an integer would be fine. An integer is a real numerical value so fits the definition of number. I'd personally support removing the word "decimal" from that paragraph.

At the same time, I'd be reluctant to specify a specific decimal format, e.g., limiting to 2 or 3 decimal places. For example, if the range were 0.01 to 0.05, then arbetrarily limiting the number to 2 or 3 decimal places would not be good.

If the primary concern is screen reader verbosity, we could potentially add some guidance to the APG that alerts people to the ramifications of how the value of aria-valuenow is formatted. Another path is to address the problem with screen reader developers.

**Comment by jnurthen** [comment]
IMO this is a screen reader developer issue - giving a % as anything other than an integer in these cases is not useful. Anyone volunteer to log bugs against screen readers?

**Comment by jnurthen** [comment]
I don't see anything to do in the spec here anyway.... Number seems correct to me as it covers decimals and integers and both are allowed. 
I'm going to close this but feel free to reopen if anything needs to change in the spec language
